src/main/resources/plugin-additional-permissions.yml

```diff
@@ -0,0 +1,21 @@
+cluster_permissions:
+  - "indices:data/read*"
+  - "indices:data/write/index*"
+  - "cluster:admin/opendistro/ad*"
+  - "cluster:admin/plugin/forecast*"
+index_permissions:
+  - index_patterns:
+      - ".opendistro-anomaly-detectors"
+      - ".opendistro-anomaly-results*"
+      - ".opendistro-anomaly-checkpoints"
+      - ".opendistro-anomaly-detection-state"
+      - ".opendistro-anomaly-detector-jobs"
+      - "opensearch-ad-plugin-result*"
```

Comment:
opensearch-ad-plugin-result* is not protected index.

Reply:
There is work in flight to address this: https://github.com/opensearch-project/security/pull/5656. And this file will be updated once we merge.